fabric-registry-sync-v0/src/testmod/java/net/fabricmc/fabric/test/registry/sync/RegistrySyncTest.java

```diff
@@ -106,9 +106,9 @@ public void onInitialize() {
 
 		// Vanilla status effects don't have an entry for the int id 0, test we can handle this.
 		RegistryAttributeHolder.get(Registries.STATUS_EFFECT).addAttribute(RegistryAttribute.MODDED);
-		Registry.register(Registries.STATUS_EFFECT, new Identifier("test", "status_effect"),
-				StatusEffectAccessor.createNewStatusEffect(StatusEffectCategory.NEUTRAL, 0xffff9900)
-		);
+		//Registry.register(Registries.STATUS_EFFECT, new Identifier("test", "status_effect"),
```

Comment:
why?

Reply:
I'm looking at it now and I have no clue why that happened—my guess is the checkstyle autoapplier did it? I'll revert it in a bit.

Reply:
Fixed as of latest commit (80ec887)